### ISSUE TITLE
PSSH box generator is double hex encoding the content id

### ISSUE THREAD

**Issue by ElCap13** [emarsden/pssh-box-wasm#2]
using the generator at https://emarsden.github.io/pssh-box-wasm/generate/ if i give the following values:
key_id (hex): `00000000000000000000000000000001`
content_id (hex): `64756d6d795f636f6e74656e745f6964`

it generates this pssh: `AAAAanBzc2gBAAAA7e+LqXnWSs6jyCfc1R0h7QAAAAEAAAAAAAAAAAAAAAAAAAABAAAANhIQAAAAAAAAAAAAAAAAAAAAASIgNjQ3NTZkNmQ3OTVmNjM2ZjZlNzQ2NTZlNzQ1ZjY5NjQyAA==`

when i decode that pssh, it gives me the following pssh data
```
WidevinePsshData<
   keyid: 00000000000000000000000000000001,
   content_id: 3634373536643664373935663633366636653734363536653734356636393634
```
the content id above is in hex and is not the same as the input.

This is all ok if the generator requires the content id in text format but if i try inputting the content id in text format, i get an error saying `Error: invalid hex encoding: content_id`

**Comment by emarsden** [comment]
You are right, the tool was decoding the hex input twice, should now be fixed.

Thanks.
